src/runtime_src/core/include/xrt/xrt_graph.h

```diff
@@ -28,6 +28,7 @@
 # include <chrono>
 # include <string>
 # include <cstdint>
+# include "experimental/xrt_hw_context.h"
 #endif
 
 typedef void *xrtGraphHandle;
```

Comment:
Do we need this typedef?

Reply:
there are public API functions such as "xrtGraphOpen", "xrtGraphOpenExclusive" etc. are still using this handle